### PR TITLE
[Filebeat] Upgrade Postgres module to work with version 11

### PR DESCRIPTION
**Pull request by sayden**
Postgres module is being only tested with 9.6 version. In this PR, log lines from Postgres 11.4 have been introduced.

At the same time, the pipeline has needed a slight change to allow parsing of error codes (it seems that they are visible by default in Postgres 11, althought I couldn't confirm this reading docs

### CODE REVIEW

**Comment by sayden** [comment]
`error.code` field is detected as failed test but, in fact, it's an ECS field. Blocked until https://github.com/elastic/beats/issues/13052 is solved

File: filebeat/tests/system/test_modules.py [hunk 1]
```diff
@@ -162,8 +162,9 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
             assert obj["event"]["module"] == module, "expected event.module={} but got {}".format(
                 module, obj["event"]["module"])
 
-            assert "error" not in obj, "not error expected but got: {}".format(
-                obj)
+            if "error" in obj:
+                assert "message" not in obj["error"], "not error message expected but got: {}".format(
+                    obj)
```
[the remark this answers is not in the document]
Comment (sayden) [reply filebeat/tests/system/test_modules.py]: Good catch! Thanks!

File: filebeat/module/postgresql/log/test/postgresql-11.4.log-expected.json [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+    {
+        "@timestamp": "2019-07-23T12:06:24.406Z",
+        "event.dataset": "postgresql.log",
+        "event.module": "postgresql",
+        "event.timezone": "UTC",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.level": "LOG",
+        "log.offset": 0,
+        "message": ": database system was interrupted; last known up at 2019-07-23 12:03:20 UTC",
```
[the remark this answers is not in the document]
Comment (sayden) [reply filebeat/module/postgresql/log/test/postgresql-11.4.log-expected.json]: Fixed! Good catch

File: filebeat/module/postgresql/log/test/postgresql-11.4.log [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+2019-07-23 12:06:24.507 UTC [1] LOCATION:  reaper, postmaster.c:2889
+2019-07-23 12:06:30.536 UTC [44] LOG:  00000: connection received: host=[local]
+2019-07-23 12:06:30.536 UTC [44] LOCATION:  BackendInitialize, postmaster.c:4206
+2019-07-23 12:06:30.537 UTC [44] LOG:  00000: connection authorized: user=postgres database=postgres
+2019-07-23 12:06:30.537 UTC [44] LOCATION:  PerformAuthentication, postinit.c:279
+2019-07-23 12:06:33.732 UTC [44] LOG:  00000: statement: show config_filel;
+2019-07-23 12:06:33.732 UTC [44] LOCATION:  exec_simple_query, postgres.c:975
+2019-07-23 12:06:33.732 UTC [44] ERROR:  42704: unrecognized configuration parameter "config_filel"
+2019-07-23 12:06:33.732 UTC [44] LOCATION:  GetConfigOptionByName, guc.c:8342
+2019-07-23 12:06:33.732 UTC [44] STATEMENT:  show config_filel;
```
[the remark this answers is not in the document]
Comment (sayden) [reply filebeat/module/postgresql/log/test/postgresql-11.4.log]: Let me take a look at it. Maybe it's not so difficult

[the remark this answers is not in the document]
Comment (jsoriano) [reply filebeat/module/postgresql/log/test/postgresql-11.4.log]: But let's leave it for another PR in any case :slightly_smiling_face:

**Pull request merged**